config/initializers/devise.rb

```diff
@@ -158,13 +158,15 @@ def session_cookie
   # given strategies, for example, `config.params_authenticatable = [:database]` will
   # enable it only for database (email + password) authentication.
   # config.params_authenticatable = true
+  config.params_authenticatable = [:pam, :database]
 
   # Tell if authentication through HTTP Auth is enabled. False by default.
   # It can be set to an array that will enable http authentication only for the
   # given strategies, for example, `config.http_authenticatable = [:database]` will
   # enable it only for database authentication. The supported strategies are:
   # :database      = Support basic authentication with authentication key + password
-  config.http_authenticatable = [:pam, :database]
```

Comment:
@devkral what was the reasoning behind enabling HTTP Basic Authentication with Devise? As far as I can tell we've never really supported it? There's no documentation for PAM Authentication nor how you're using it, so it's unclear to me why this was needed?